assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js

```diff
@@ -87,15 +88,33 @@ function DashboardClicksWidget() {
 	if ( ! data || ! data.length ) {
 		return <ReportZero moduleSlug="search-console" />;
 	}
+	// Split the data in two chunks.
+	const half = Math.floor( data.length / 2 );
+	const latestData = data.slice( half );
+	const olderData = data.slice( 0, half );
 
-	const { totalClicks, totalClicksChange, dataMap } = extractSearchConsoleDashboardData( data );
+	const totalClicks = sumObjectListValue( latestData, 'clicks' );
+	const totalOlderClicks = sumObjectListValue( olderData, 'clicks' );
+	const totalClicksChange = changeToPercent( totalOlderClicks, totalClicks );
+
+	const sparklineData = [
+		[
+			{ type: 'string', label: 'Day' },
+			{ type: 'number', label: 'Clicks' },
+		],
+		...extractForSparkline( latestData, 'clicks', 'keys.0' ).map( ( row ) => {
+			const date = new Date( row[ 0 ] );
+			// Sparkline data needs headers and dates formatted as MM/DD
+			return [ date.getMonth() + 1 + '/' + date.getUTCDate(), row[ 1 ] ];
```

Comment:
This is pretty minor but using string interpolation would better-signal what this item in the array is meant to be:

```suggestion
			return [ `${ date.getMonth() + 1 }/${ date.getUTCDate() }`, row[ 1 ] ];
```

I dunno, maybe it's just me, but I always prefer string interpolation over concat operations for readability.